### DISCUSSION TITLE
Consistency between passphrase and password in wording ?

### DISCUSSION THREAD

**Discussion by Cwpute** [the-djmaze/snappymail#1538] [Q&A]
Hi, i'm looking a the french translation for Snappymail, and i'm a bit confused because Snappymail sometimes uses the words `passphrase` and sometimes `password`... and the previous french translator sometimes chose to use one word for the other, and vice-versa. 😵💫
I believe the passphrase is for PGP security features only, but i find passphrase used in other parts of the translation.
Can you confirm that passphrase and password mean two different things, and that they are both used in the right settings ?

**Comment by the-djmaze** [comment]
Instead of using a short, complex password that is hard to remember, consider using a longer passphrase.

With that said, in SnappyMail i try to rename `password` to `passphrase` slowly everywhere.
`password` is just a thing from the pas(s)t that is not suitable these days.

**Reply by Cwpute** [reply]
My specific problem is that, there is no easily understandable and direct translation for `passphrase` in french, and i don't think most french users are used to using passphrases at all. Two different wordings for the same thing will have people be confused i believe.
Should we wait until all instances of `password` are removed until further translation ?

**Reply by the-djmaze** [reply]
password is derived from word (mot in French).
passphrase is like a sentence (phrase in French) and consists of multiple words.

So the translation could be more like "security phrase":
* peine de sécurité
* phrase secrète
* mots de passe
* etc.
